### PR TITLE
Fixes #28297 - rubocop updated to 0.75

### PR DESCRIPTION
**Pull request by lzap**
SSIA + I took the chance and cleaned up todo a bit.

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -1,127 +1,52 @@
+require: rubocop-rails
+
 inherit_from:
   - .rubocop_todo.yml
 
 AllCops:
+  TargetRubyVersion: 2.4
```
Comment (mmoll) [inline .rubocop.yml]: 2.3 should be supported until Debian/stretch gets dropped.

**Comment by lzap** [comment]
The error seems to be irrelevant ("SQLite3::SQLException: too many terms in compound SELECT"). Shall we drop sqlite3 from executing tests now?

**Comment by adamruzicka** [comment]
Wouldn't it be better to squash some migrations to avoid running into the issue?

**Comment by lzap** [comment]
Sorry what?

**Comment by mmoll** [comment]
I'd be all in for merging migrations, however, this is not related to this PR :)

**Comment by adamruzicka** [comment]
Indeed not, I was just reacting to this.

> Shall we drop sqlite3 from executing tests now?

**Comment by adamruzicka** [comment]
Thank you @lzap !

**Pull request merged**